src/Pages/ChannelRequestsComponents/ChannelStatusLogsTooltip.razor

```diff
@@ -32,6 +48,7 @@
             }
             text += $"<li>{emoji} {channelStatusLog.DateTime:dd/MM/yyyy HH:mm:ss} - {channelStatusLog.Description}</li>";
         }
+        text += $"<li>Double tap copy error text</li>";
```

Comment:
Could you add it outside of the unordered list?

Reply:
Fixed